templates/account/partials/login_form.html

```diff
@@ -6,10 +6,7 @@
   {% csrf_token %}
   <fieldset>
     {% bootstrap_form form %}
-    {% if redirect_field_value %}
-      <input type="hidden" name="next" name="{{ redirect_field_name }}"
-             value="{{ redirect_field_value }}"/>
-    {% endif %}
+      <input type="hidden" name="next" value="{{ next }}"/>
```

Comment:
`redirect_field_value` is used in few other templates as well so I assume it's there on purpose. It looks like it's value is not passed properly to "loging_form.html", that's why the redirect is not working.

Reply:
I might be wrong, but:
The usage of `redirect_field_value` and `redirect_field_name` came from django-allauth which was used in saleor. However since django-allauth was replaced with social_django those variables became kinda obsolete. social_django use different approach and use context processor to provide `REDIRECT_FIELD_NAME` and `REDIRECT_FIELD_VALUE`.

I would suggest replacing variables similiar to #1453 

Btw. `REDIRECT_FIELD_NAME` is actually `REDIRECT_FIELD_NAME` from django.contrib.auth